index.html

```diff
@@ -11,64 +11,66 @@
+                    <i class="fa fa-bars fa-lg"></i>
+                    <ul class="nav-links">
+                        <li>
+                            <a href="#card-section">Portfolio</a>
+                        </li>
+                        <li>
+                            <a href="#about">About</a>
+                        </li>
+                        <li>
+                            <a href="#form-section">Contact</a>
+                        </li>
+                    </ul>
+                </nav>
+            </header>
+            <section class="main-container">
+                <div class="main-content-intro">
+                    <div>
+                        <h1>I'm Moyasi. <br />
+                            Glad to see you!
+                        </h1>
+                    </div>
+                    <div>
+                        <p>
+                            I'm a software developer! I can help you build a product , feature or website Look through some
+                            of my work and experience!
+                            If you like what you see and have a project you need coded, don't hestiate to contact me.
+                        </p>
+                    </div>
+                    <div id="social__container">
+                        <a href="#" id="social-heading">Let's Connect</a>
+                            <div class="social__icons">
+                                <ul>
+                                    <li>
+                                        <a href="https://twitter.com/moyasi_ginko"><img src="asset/twitter.svg"
+                                                alt="twitter icon"></a>
+                                    </li>
+                                    <li>
+                                        <a href="https://www.linkedin.com/in/mahmudur-rahman-a8a151257/"><img
+                                                src="asset/Linkedin icon.svg" alt="Linkedin icon"></a>
+                                    </li>
+                                    <li>
+                                        <a href="#"><img src="asset/M.svg" alt="M icon"></a>
+                                    </li>
+                                    <li>
+                                        <a href="https://github.com/MoyasiGinko"><img src="asset/github.svg"
+                                                alt="github icon"></a>
+                                    </li>
+                                    <li>
+                                        <a href="#"><img src="asset/thumbs.svg" alt="thumbs icon"></a>
+                                    </li>
+                                </ul>
+                            </div>
+                    </div>
                 </div>
-            </div>
-        </section>
+            </section>
+        </div>
         <section id="card-section">
             <div class="card__container">
                 <div class="items card-1">
```

Comment:
- [ ] In your design the margin between all the description elements is less than what it should be to make it identical to the figma design. One of the main requirements is to make our design's fonts, margins and paddings identical to the figma design so kindly consider measuring the margin between those elements in the figma design and then apply that in your design.
- [ ] Also the margin between the `See Project` button and the languages is none so kindly take extra care to fix that.
your design
![as](https://user-images.githubusercontent.com/96683843/230521794-0fbb2904-2843-4f3e-86e0-f07e28c7f665.jpg)

figma design
![qw](https://user-images.githubusercontent.com/96683843/230521802-f22c1382-3e72-4b29-b7a5-1788158e1a69.jpg)

Reply:
I have fixed the issues on the `see project` button.